src/views/side_bar_panel.dot

```diff
@@ -2,7 +2,7 @@
 
   <div class="side_bar__logo"></div>
   <div class="side_bar__logo__limit"></div>
-  <div class="side_bar__result side_bar__item  {{= this.current === 0 ? 'side_bar__item--active': ''}}">
+  <div {{= click(this.toggleResult, this) }} class="side_bar__result side_bar__item {{= this.current === 0 ? 'side_bar__item--active': ''}}">
```

Comment:
Trigger "toggleResult" on this click is not enough, as the poi panel could be toggled via other actions.